sources/packages/forms/src/form-definitions/institutionprofile.json

```diff
@@ -562,140 +562,216 @@
+                  "calculateValue": "",
+                  "calculateServer": false,
+                  "allowCalculateOverride": false,
+                  "validateOn": "change",
+                  "validate": {
+                    "required": true,
+                    "onlyAvailableItems": true,
+                    "customMessage": "",
+                    "custom": "",
+                    "customPrivate": false,
+                    "json": "",
+                    "strictDateValidation": false,
+                    "multiple": false,
+                    "unique": false
+                  },
+                  "unique": false,
+                  "errorLabel": "",
+                  "errors": "",
+                  "key": "regulatingBody",
+                  "tags": [],
+                  "properties": {},
+                  "conditional": {
+                    "show": "",
+                    "when": null,
+                    "eq": "",
+                    "json": ""
+                  },
+                  "customConditional": "",
+                  "logic": [],
+                  "attributes": {
+                    "data-cy": "regulatingBody"
+                  },
+                  "overlay": {
+                    "style": "",
+                    "page": "",
+                    "left": "",
+                    "top": "",
+                    "width": "",
+                    "height": ""
+                  },
+                  "type": "select",
+                  "indexeddb": {
+                    "filter": {}
+                  },
+                  "selectFields": "",
+                  "searchField": "",
+                  "searchDebounce": 0.3,
+                  "minSearch": 0,
+                  "filter": "",
+                  "redrawOn": "",
+                  "input": true,
+                  "searchThreshold": 0.3,
+                  "prefix": "",
+                  "suffix": "",
+                  "dataGridLabel": false,
+                  "showCharCount": false,
+                  "showWordCount": false,
+                  "allowMultipleMasks": false,
+                  "addons": [],
+                  "lazyLoad": true,
+                  "authenticate": false,
+                  "ignoreCache": false,
+                  "fuseOptions": {
+                    "include": "score",
+                    "threshold": 0.3
+                  },
+                  "id": "e5eqny",
+                  "defaultValue": ""
+                }
+              ],
+              "width": 6,
+              "offset": 0,
+              "push": 0,
+              "pull": 0
+            },
+            {
+              "components": [
+                {
+                  "autofocus": false,
+                  "input": true,
+                  "tableView": true,
+                  "inputType": "text",
+                  "inputMask": "",
+                  "label": "Other Institution Regulating Body",
```

Comment:
Looking at the other fields on the form it looks like the case should be "Other institution regulating body".